packages/eslint-config/index.js

```diff
@@ -40,6 +40,7 @@ module.exports = {
     'import/extensions': 'off',
     'es/no-regexp-lookbehind-assertions': 'error',
     curly: ['error'],
+    'prefer-arrow-callback': 'off',
```

Comment:
@mattrunyon had a great suggestion to just enable named functions instead of disabling the rule entirely (https://eslint.org/docs/rules/prefer-arrow-callback#allownamedfunctions):
```suggestion
    'prefer-arrow-callback': ['error', { allowNamedFunctions: true }],
```